src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -310,3 +310,80 @@ export interface DataHealthGetTrackingIdsResponse {
      */
     values: DataHealthFacetValue[];
 }
+
+export interface DataHealthGetEventProblemsParams extends OrganizationParamParts, TimeRangeParamParts, Paginated {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType?: string[];
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub?: string[];
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId?: string[];
+}
+export interface EventProblemsResponse extends PaginatedResponse {
+    /**
+     * List of the event problems.
+     */
+    problems: EventProblemsResponseItem[];
+}
+
+export interface EventProblemsResponseItem {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType: string;
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId: string;
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub: string;
+    /**
+     * Number of failed events.
+     */
+    failedEvents: number;
+    /**
+     * Number of total events.
```

Comment:
```suggestion
     * The total number of events.
```

what does this mean though? Is it the total number of events that match the filters, or is it the total number of events with problems that match the filters? This should be specified in the documentation.

Reply:
It's the total number of failed events (events with validation error), I think it should say something like `The total number of failed events.`

Reply:
```suggestion
     * The total number of failed events.
```